src/mcp/state.py

```
@@ -30,164 +69,295 @@
+            )
+    return result
+
+
+def _idf_values(idf: IDF, *object_types: str) -> list[Any]:
+    values: list[Any] = []
+    seen: set[int] = set()
+    for object_type in object_types:
+        try:
+            objs = idf.all_of_type(object_type)
+        except Exception:
+            continue
```

Ruff (0.16.0)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)